### PR TITLE
fix typo in issue template

### PR DESCRIPTION
**Pull request by FahimFBA**
Closing #7743 

* Do the tests still pass? (https://docs.wagtail.io/en/latest/contributing/developing.html#testing) - YES
* Does the code comply with the style guide? (Run `make lint` from the Wagtail root) - YES
* For Python changes: Have you added tests to cover the new/fixed behaviour? - YES
* For front-end changes: Did you test on all of [Wagtail’s supported environments](https://docs.wagtail.io/en/latest/contributing/developing.html#browser-and-device-support)?
    * **Please list the exact browser and operating system versions you tested**. Chrome and Windows 11
    * **Please list which assistive technologies you tested**.
* For new features: Has the documentation been updated accordingly? - YES

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,4 +121,4 @@ We thank the following organisations for their services used in Wagtail's develo
 [![Version](https://img.shields.io/pypi/v/wagtail.svg)](https://pypi.python.org/pypi/wagtail/) 
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/wagtail/wagtail.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/wagtail/wagtail/alerts/)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/wagtail/wagtail.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/wagtail/wagtail/context:python)
-[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/wagtail/wagtail.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/wagtail/wagtail/context:javascript)
+[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/wagtail/wagtail.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/wagtail/wagtail/context:javascript)
```
Comment (lb-) [inline README.md:124]: I will merge in but looks like this change was unintentional so will remove from the final commit.

**Review by lb- — APPROVED** [review]
Thanks for fixing, we will merge this in.

**Comment by lb-** [comment]
Merged in via 1cbd3c2616c51bbc2a7473f6a12a760529e210bc

File: README.md (hunk 1 above)
Comment (FahimFBA) [reply README.md:124]: Yes, this part was unintentional. Thanks for understanding that.

**Pull request closed without merging**